src/gcp/cloudfunctions.js

```diff
@@ -20,7 +19,8 @@ function _functionsOpLogReject(func, type, err) {
   } else {
     logger.info(err.message);
   }
-  return Promise.reject(err.message);
+  err.context = { function: func };
+  return Promise.reject(err);
```

Comment:
I feel like I'm being a stickler, or pedantic, but doing this blows away all the context of the original error. `FirebaseError` (from `../error`) handles this nicely:

```javascript
new FirebaseError(
  `Failed to ${type} function "${func}"`,
  {
    original: err,
    context: { function: func },
  }
)
```

Reply:
Done